km61/Servatmand_Mariam/laboratory1.sql

```diff
@@ -1,2 +1,129 @@
 -- LABORATORY WORK 1
 -- BY Servatmand_Mariam
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+робити вибірки з таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Servatmand INDENTIFIED BY Servatmand
+DEFAULT TABLESPACE USERS
+TEMPORARY TABLESPACE TEMP
+
+ALTER Servatmand QUOTA 100 MB ON USERS
+ GRANT "CONNECT" TO Servatmand
+GRANT SELECT ANY TABLE TO USER
```

Comment:
Use semicolon ; in the of command.
Use name of the user, instead of USER.
`GRANT "CONNECT" TO Servatmand; GRANT SELECT ANY TABLE TO Servatmand;`